scripts/run_lighthouse_tests.py

```diff
@@ -189,13 +193,14 @@ def main(args: Optional[List[str]] = None) -> None:
         lighthouse_mode = LIGHTHOUSE_MODE_PERFORMANCE
         server_mode = SERVER_MODE_PROD
 
-    if lighthouse_mode == LIGHTHOUSE_MODE_PERFORMANCE:
-        print('Building files in production mode.')
-        build.main(args=['--prod_env'])
-    else:
+    if (lighthouse_mode == LIGHTHOUSE_MODE_ACCESSIBILITY or
+        parsed_args.skip_build == 'true'):
         build.main(args=[])
         common.run_ng_compilation()
         run_webpack_compilation()
+    else:
+        print('Building files in production mode.')
+        build.main(args=['--prod_env'])
```

Comment:
Can we keep `run_webpack_compilation()` as the default mode?

Modify the logic to:
```
if LIGHTHOUSE_MODE_PERFORMANCE is true:
    // prod env
else:
   common.run....
   if not skip_build:
      // run webpack
```

Reply:
do you mean we should use accessibility mode when skip_build is true? and we cannot skip run_webpack_compilation in accessibility mode or build would fail.

Reply:
No, we run accessibility mode when `LIGHTHOUSE_MODE_PERFORMANCE` is false, and the `skip_build` option should skip the webpack build step. 

Just to clarify, which is the webpack build step?

Reply:
I think we build it in build.main when we pass --prod_env as argument. This run_webpack_compilation will be excuted anyway.
https://github.com/oppia/oppia/blob/ac3da93514da9472b553b911e7417d0a7031f4ff/scripts/build.py#L1437

Reply:
Sorry about the confusion. Let's modify the performance mode so that if the `skip_build` option is used the webpack build step is skipped.

Reply:
done.